rollup-tests.config.js

```diff
@@ -0,0 +1,39 @@
+import * as glob from 'glob';
+import { babel } from '@rollup/plugin-babel';
+import commonjs from '@rollup/plugin-commonjs';
+import json from '@rollup/plugin-json';
+import multi from '@rollup/plugin-multi-entry';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
```

Comment:
This is a pretty standard set of plugins. I think most of them speak for themselves.

- @rollup/plugin-commonjs allows the use of CommonJS-based dependencies. This can be removed if all dependencies are ES module based
- @rollup/plugin-multi-entry supports bundles with multiple entry points. This is mainly useful for tests
- @rollup/plugin-node-resolve makes `import` statements resolve module specifies the same way as Node (and Browserify) do, which is different than browsers (eg. by not requiring the `.js` extension and by doing lookups in node_modules)